### DISCUSSION TITLE
Jotai Devtools Crashes in my project and works in other projects

### DISCUSSION THREAD

**Discussion by legions-developer** [pmndrs/jotai#2630] [Bug report]
### Bug Description

`"use client";

import { Provider, createStore } from "jotai";
import { DevTools } from "jotai-devtools";
import "jotai-devtools/styles.css";
import { ReactNode } from "react";

export const jotaiStore = createStore();

export function JotaiProvider({ children }: { children: ReactNode }) {
  return (
    <Provider store={jotaiStore}>
      <DevTools store={jotaiStore} />
      {children}
    </Provider>
  );
}
`

im using this format of code for my jotai dev tools but when i use jotai dev tools then my nextjs server gets crashed without any error shown in terminal

### Check List

- [X] I've added a link to a [TypeScript Playground](https://www.typescriptlang.org/play), a [StackBlitz Project](https://stackblitz.com/) or something else with a minimal reproduction.

**Comment by himself65** [comment]
What is the full code?
